sklearn/tests/test_docstrings.py

```diff
@@ -39,7 +39,7 @@
     "sklearn.metrics.cluster._supervised.adjusted_rand_score",
     "sklearn.metrics.cluster._supervised.entropy",
     "sklearn.metrics.cluster._supervised.fowlkes_mallows_score",
-    "sklearn.metrics.cluster._supervised.rand_score",
+    "sklearn.metrics.cluster._supervised.v_measure_score",
```

Comment:
Is this a spurious change?
```suggestion
```

Reply:
Yes totally, I had a conflict after merging main, and I thought `v_measure_score` wasn't merged yet. Sorry about that.